node/task.ts

```diff
@@ -1799,7 +1799,7 @@ export class TestPublisher {
         }
 
         if (resultFiles) {
-            properties['resultFiles'] = resultFiles;
+            properties['resultFiles'] = Array.isArray(resultFiles) ? resultFiles.join() : resultFiles;
```

Comment:
Are you sure this won't introduce any unexpected changes?
Correct me if I'm wrong, but seems that with this change [this line in taskcommand](https://github.com/microsoft/azure-pipelines-task-lib/blob/f325a44515b6852b29d5de65af7666e77c84333b/node/taskcommand.ts#L37) will work differently

Reply:
This line you've mentioned handles arrays the same way as strings except the case with an empty string or array. This change is introduced to make it handled the same way in any case. If an array is passed it would be 'joined' with a standard separator.